MiniApp/Classes/MiniAppLister.swift

```diff
@@ -14,4 +14,21 @@ internal class MiniAppLister {
             }
         }
     }
+
+    func fetchMiniApp(miniAppId: String, apiClient: MiniAppClient, completionHandler: @escaping (Result<MiniAppInfo, Error>) -> Void) {
```

Comment:
`fetchMiniApp` sounds like retrieving the complete Mini App whereas the function is about getting the metadata. 
On Android side, we are have evolved the `List` module into `AppInfo`.

Reply:
You can refer: https://github.com/rakutentech/android-miniapp/pull/45/files

Reply:
OK, sounds good to me